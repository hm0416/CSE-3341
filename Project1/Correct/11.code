program
	int x;
begin
	x=1*2*3*4;
	output x;
end