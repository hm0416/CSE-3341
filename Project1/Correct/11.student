WHILE
NEGATION
LPAREN
ID[x]
ADD
ID[y]
ASSIGN
ASSIGN
ASSIGN
ASSIGN
MULT
MULT
CONST[101]
BEGIN
ID[test]
SEMICOLON
ID[test]
ASSIGN
SUB
SUB
SEMICOLON
SEMICOLON
ENDWHILE
END

