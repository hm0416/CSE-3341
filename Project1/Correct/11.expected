WHILE
NEGATION
LPAREN
ID[x]
ADD
ID[y]
ASSIGN
ASSIGN
EQUAL
ID[z10]
MULT
CONST[101]
BEGIN
ID[test]
SEMICOLON
ID[test]
ASSIGN
ID[test1]
SUB
ID[test2]
SEMICOLON
ENDWHILE
END
